github/github.go

```diff
@@ -143,6 +146,7 @@ type Client struct {
 	Licenses       *LicensesService
 	Migrations     *MigrationService
 	Reactions      *ReactionsService
+	Marketplace    *MarketplaceService
```

Comment:
Let's also move this one two lines up to keep this list sorted.

Reply:
Have no problem moving it up, but note that there are items higher in the list which are not sorted.

Reply:
Would you mind sorting them while you are here, please?

Reply:
Done